Ruby/wc/file_content.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+# class to provide file utility information.
+class FileContent
+  def initialize(file_content)
+    @file_content = file_content
+  end
+
+  def size
+    @file_content.bytesize
+  end
+
+  def lines_count
+    lines = @file_content.lines
+    lines.count
+  end
+
+  def word_count
+    @file_content.split(/\s+/).length
+  end
+
+  def char_count
+    @file_content.length
+  end
```

Comment:
NIT: I guessing all these functions assume that a string will be used to create this object. Should there be checks against passing anything else? Don't even know how it'd need to be done 😅

Reply:
Or may be  this can be named TextFileContent 🤔

Reply:
I think I'm reading [everything as a string](https://github.com/aganapuram/CodingChallengesJC/pull/1/files#diff-84e265f1412bfd60c646a99e7ebfea2fee7db76069b1cdd9009c2dca1a8d178bR29-R32) from the file. I don't think there's a need for other checks.

Reply:
yeah  - I was thinking we should explicitly reject anything other than .txt file 🤷🏼 
Out of scope - 
Curious, what if an mp3 file was supplied as input? - I'm guessing Ruby will try to interpret the data in the file as String utf-8 or some form of this encoding and try to run calculations